### ISSUE TITLE
Serialization: extrude, and altitudeMode incorrectly positioned?

### ISSUE THREAD

**Issue by blipmusic** [georust/kml#24]
When I serialize `KmlDocument` (with `version: kml::KmlVersion::V22`) I get e.g.:

```
<Point>
	<coordinates>13.40984,52.51778</coordinates>
	<altitudeMode>clampToGround</altitudeMode>
	<extrude>0</extrude>
</Point>
```

which fails to validate against the KML v2.2 schema. It seems `altitudeMode` and `extrude` are incorrectly positioned, they should precede `coordinates`, i.e.:

```
<Point>
	<extrude>0</extrude>
	<altitudeMode>clampToGround</altitudeMode>
	<coordinates>13.40984,52.51778</coordinates>
</Point>
```

This validates against the v2.2 schema. (c.f. lines 885-887 in http://schemas.opengis.net/kml/2.2.0/ogckml22.xsd).

Would changing the order [here](https://github.com/georust/kml/blob/e7ef93afbeb88addb52a0b22ce6633d8b4fbff7b/src/writer.rs#L130) be sufficient? (sorry still new to pull requests etc)

**Comment by pjsier** [comment]
@blipmusic thanks for this issue! I hadn't taken a close look at the sequence restrictions since initially this was mostly focused on reading. Changing the order would fix the issue, feel free to open a PR and I'm happy to answer any questions if you run into any issues

**Comment by blipmusic** [comment]
And thanks for your work on this!

I forked the main branch [here](https://github.com/blipmusic/kml) and simply changed write order for extrude, and altitudeMode for [write_point](https://github.com/blipmusic/kml/blob/b79627c1662e6cddd84fea75272445ba24142ffd/src/writer.rs#L130) and extrude, tessellate, altitudeModeGroup for [write_geom_props](https://github.com/blipmusic/kml/blob/b79627c1662e6cddd84fea75272445ba24142ffd/src/writer.rs#L441). (made a mess of commits since I forgot return values changed...)

This validates against v2.2 so far.

So doing the same for this repo, should both of these go into the same pull request? Also against which branch, a new or main?

**Comment by blipmusic** [comment]
I made a PR, but eh I accidentally merged your other suggestion for child elements in Placemark, sorry! I reverted that before making the request. Hope you'll be able to squash the commits it into some cleaner! :)

**Comment by pjsier** [comment]
Closed by #25, thanks for pointing this out and for putting together the fix!
